aidants_connect_habilitation/templates/aidants_connect_habilitation/admin/organisation_request/change_form.html

```diff
@@ -4,7 +4,7 @@
 {% block object-tools-items %}
   {% with status=adminform.form.instance.status %}
 
-      {% if status in "AC_VALIDATION_PROCESSING,CHANGES_REQUESTED,VALIDATED,REFUSED,CLOSED" %}
+      {% if status in "AC_VALIDATION_PROCESSING,CHANGES_REQUIRED,VALIDATED,REFUSED,CLOSED" %}
```

Comment:
J'approuve comme ça mais il me semble qu'une manière de rendre le code pluscorrect serait de faire ça :

```python
# Dans aidants_connect_common/utils/constants.py
class RequestStatusConstants(TextChoicesEnum):
    ...
    def trouver_un_nom_ici(self) -> Set[TextChoicesEnum]:
        return {
            self.AC_VALIDATION_PROCESSING,
            self.CHANGES_REQUIRED,
            self.VALIDATED,
            self.REFUSED,
            self.CLOSED,
        }
passer `RequestStatusConstants` dans le contexte et faire :

```html
# dans aidants_connect_web/templates/aidants_connect_web/admin/carte_totp/change_form.html
      {% if status in "RequestStatusConstants.trouver_un_nom_ici" %}
```

Reply:
je vais merge et je regarderais pour pas mettre la liste dans le template oui